packages/fluent/scss/tabstrip/_variables.scss

```diff
@@ -0,0 +1,145 @@
+/// The border color of the tabstrip ripple
+/// @group tabstrip
+$kendo-tabstrip-item-ripple-border: get-theme-color-var( primary-130 ) !default;
+/// The border radius of the tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-border-radius: var( --kendo-border-radius-md, 0 ) !default;
+/// The gap between the tabstrip items
+/// @group tabstrip
+$kendo-tabstrip-item-gap: map-get( $kendo-spacing, 2 ) !default;
+///  The background color of the tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-bg: transparent !default;
+/// The text color of the tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-text: var( --kendo-component-text, initial ) !default;
+/// The border color of the tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-border: transparent !default;
+/// The text color of the disabled tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-disabled-text: var( --kendo-disabled-text, initial ) !default;
+/// The background color of the disabled tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-disabled-bg: none !default;
+/// The border color of the disabled tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-disabled-border: var( --kendo-disabled-border, initial ) !default;
+
+/// The background color of hovered tabs
+/// @group tabstrip
+$kendo-tabstrip-item-hovered-bg: get-theme-color-var( neutral-20 ) !default;
+/// The text color of hovered tabs
+/// @group tabstrip
+$kendo-tabstrip-item-hovered-text: get-theme-color-var( neutral-190 ) !default;
+/// The border color of hovered tabs
+/// @group tabstrip
+$kendo-tabstrip-item-hovered-border: get-theme-color-var( neutral-20 ) !default;
+/// The background color of selected tabs
+/// @group tabstrip
+$kendo-tabstrip-item-selected-bg: var( --kendo-component-bg, transparent ) !default;
+/// The text color of selected tabs
+/// @group tabstrip
+$kendo-tabstrip-item-selected-text: get-theme-color-var( neutral-190 ) !default;
+/// The border color of selected tabs
+/// @group tabstrip
+$kendo-tabstrip-item-selected-border: get-theme-color-var( primary-130 ) !default;
+/// The background gradient of selected tabs
+/// @group tabstrip
+$kendo-tabstrip-item-selected-font-weight: var( --kendo-font-weight-bold, normal ) !default;
+/// The focus shadow of the tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-focused-shadow: $kendo-list-item-focus-shadow !default;
+/// The text color of the sortable tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-dragging-text: get-theme-color-var( primary-130 ) !default;
+
+// Indicator
+$kendo-tabstrip-indicator-size: null !default;
+$kendo-tabstrip-indicator-color: null !default;
+
+/// The horizontal padding of tabstrip content
+/// @group tabstrip
+$kendo-tabstrip-content-padding-x: map-get( $kendo-spacing, 2 ) !default;
+/// The vertical padding of tabstrip content
+/// @group tabstrip
+$kendo-tabstrip-content-padding-y: map-get( $kendo-spacing, 3 ) !default;
```

Comment:
According to the design this should be 6px (`map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, thin )`)